vim25/types/helpers.go

```diff
@@ -75,6 +75,14 @@ func (r *ManagedObjectReference) FromString(o string) bool {
 	return true
 }
 
+func (r ManagedObjectReference) Equal(b ManagedObjectReference) bool {
+	return r.Type == b.Type && r.Value == b.Value
+}
+
+func (r ManagedObjectReference) EqualStrict(b ManagedObjectReference) bool {
```

Comment:
Trying to find where `EqualStrict` and `Equal` are used, are these interface funcs used by an external package or ?